src/components/Notice/Notice.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint react/no-array-index-key: 0 */
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Alert, Button, Container } from 'react-bootstrap'
+
+const Notice = ({ alert, buttonProps, dismissible, withBackButton }) => {
+  const [show, setShow] = useState(true)
+  const { title, body, variant } = alert
+  let onClick, text
+  if (withBackButton) ({ onClick, text } = buttonProps)
+
+  return (
+    show && (
+      <Container>
+        <Alert className='my-5 text-break' variant={variant} onClose={() => setShow(false)} dismissible={dismissible}>
+          {title && (
+            <Alert.Heading>{title}</Alert.Heading>
+          )}
+          {body.map((text, index) => (
+            <p className='mb-0' key={index}>{text}</p>
+          ))}
+          {withBackButton && (
+            <>
+              <hr />
+              <div className='d-flex justify-content-end'>
+                <Button onClick={onClick} variant={`outline-${variant}`}>
+                  {text}
+                </Button>
+              </div>
+            </>
+          )}
+        </Alert>
+      </Container>
+    )
+  )
+}
+
+Notice.propTypes = {
+  alert: PropTypes.shape({
+    body: PropTypes.arrayOf(PropTypes.string).isRequired,
+    title: PropTypes.string,
+    variant: PropTypes.string.isRequired,
+  }).isRequired,
+  buttonProps: PropTypes.shape({
+    onClick: PropTypes.func,
+    text: PropTypes.string,
+  }),
+  dismissible: PropTypes.bool,
+  withBackButton: PropTypes.bool,
+}
+
+Notice.defaultProps = {
+  buttonProps: {
+    text: 'Click to return to the previous page.'
+  },
+  dismissible: true,
+  withBackButton: false,
```

Comment:
i see it is the default.. but should the default text say something else?

Reply:
I was trying to not have to write that in all the places where we have an error since the error is the most common use case. for now. lol.
but long term....it's prolly better to not have that be the default. update incoming!